### DISCUSSION TITLE
GPU Memory Requirements for Training Large-scale se_atten Models

### DISCUSSION THREAD

**Discussion by yanjin-tech** [deepmodeling/deepmd-kit#3902] [General]
Hello,

I would like to inquire about the pre-trained model used in the development of se_atten. Specifically, on what GPU with how much memory was it trained?

Recently, I have been working on a model with a very large dataset. Due to the variety in structural elements, space groups, and the number of atoms, the memory required for training data is significantly larger compared to training a model with a fixed structure. I often encounter memory error issues, and therefore, I am wondering if my problem is due to insufficient computational resources.

**Comment by njzjz** [comment]
There are some benchmark results in our published paper.
